config/routes.rb

```diff
@@ -1,6 +1,12 @@
+require 'sidekiq/web'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.